virttest/utils_v2v.py

```diff
@@ -606,7 +607,8 @@ def get_windows_event_info(self):
         """
         Get windows event log info about WSH.
         """
-        cmd = "CSCRIPT C:\WINDOWS\system32\eventquery.vbs /l application /Fi \"Source eq WSH\""
+        cmd = "CSCRIPT C:\%s\system32\eventquery.vbs /l application /Fi \"Source eq WSH\"" % \
+              self.windows_path
```

Comment:
It's better to avoid use  '\' to split a long.  You can using the following method:
```
cmd = "..."
cmd += "..."
```